doc/WebRTC.xml

```diff
@@ -498,6 +498,93 @@
         affected. However, it's recommended to keep the WebSocket session to the server open to be
         able to receive event messages from the signaling server.</para>
     </section>
+    <section xml:id="section_error_notification">
+      <title>error</title>

```

Comment:
As per this PR, After establishing a peer-to-peer WebRTC channel, both peers should remain connected to the signaling server to receive notifications from the other peer.

Reply:
Can you point me toward which sentence told you this? To be sure, this was not the intent we had, the goal here is to provide a way to send feedback if needed and possible, if the client is gone then no notification will be received and it should not be relied upon.
The goal of the notification is mainly to handle unexpected scenarios during the handshake. Once the handshake is completed, this notification should not be used, and is not mandatory to be received

Reply:
Below two example use cases are given in the PR description.

Here the first use case talks about signaling server notification to the peer saying counterpart is disconnected. It means both the peers should be connected to signaling server to receive these notifications from signaling server about it's counter part.

Examples:
1. If a peer has disconnected from the signaling server, the other peer will have to wait for an indefinite period of time before closing its session. With this proposal, the signaling server will be able to notify the connected peer that its counterpart has disconnected and the session can be closed.

2. If a client sends a connect to a device that does not have sufficient resources to start a new session, the device will now be able to send an error 1001 to inform the client that it cannot continue with this session.

Reply:
Client and Device must have a websocket connection with the signaling server for any message to be exchanged between peers. In the case a client has sent a connect request for a specific device, if that device has disconnected after the sessionId has already been provided to the client, then the signaling would send the disconnect notification. The client is already connected, so it does not add any new requirement on it. Similarly, a device should always have an active websocket connection with the signaling, otherwise the device can never receive any connect message.

Reply:
Oh ok I see the confusion around the "peer has disconnected" point. The wording should be "peer has disconnected in the middle of the handshake" just to make it clearer